tests/integration/kubernetes/runtimeclass_workloads/pod-manifest-v1.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2024 Microsoft
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginxhttps
+  
+spec:
+  runtimeClassName: kata
+  terminationGracePeriodSeconds: 0
+  containers:
+    - name: nginxhttps
+      image: "docker.io/ymqytw/nginxhttps:1.5"
```

Comment:
I am not sure docker.io is acceptable for CI testing. Let's see if more experienced reviewers have better ideas here - I will ping them after we agree on the rest of the changes.

Looks like pod-file-volume.yaml is using docker too - so maybe it is acceptable.

Reply:
@GabyCT @fidencio , do have to avoid CI testing using a container from docker.io?

The only similar test seems to be pod-file-volume.yaml (a.k.a. k8s-file-volume.bats). I noticed that most other tests are using images from quay.io.

Thanks!